src/main/java/teammates/storage/search/SearchQuery.java

```diff
@@ -40,14 +43,31 @@ public int getFilterSize(){
     }
     
     protected SearchQuery setTextFilter(String textField, String queryString){
-        String sanitizedQueryString = Sanitizer.sanitizeForSearch(queryString).toLowerCase().trim();
+        String sanitizedQueryString;
+        if(isValidEmailAddress(queryString)){
```

Comment:
Yes I notice that the other `if`s do not follow the convention. It's fine, we can refactor everything together in a separate issue.